src/js/otus/types.ts

```diff
@@ -87,7 +87,7 @@ export type OTU = OTUMinimal & {
     issues?: { [key: string]: any } | boolean;
     last_indexed_version?: number;
     most_recent_change: HistoryNested;
-    otu_schema: Array<OTUSegment>;
+    schema: Array<OTUSegment>;
```

Comment:
https://github.com/virtool/virtool-core/blob/main/virtool_core/models/otu.py

You check that this change makes sense? It conflicts with the models in core.